app/controllers/hub/efile_submissions_controller.rb

```diff
@@ -15,7 +15,7 @@ def index
     # loops through the tax_returns that have efile_submissions.
     def show
       @client = Client.find(params[:id])
-      @tax_returns = @client.tax_returns.joins(:efile_submissions) # get all tax returns with submissions
+      @tax_returns = @client.tax_returns.joins(:efile_submissions).uniq # get all tax returns with submissions
```

Comment:
`.uniq` is an in-memory operation, so this will still retrieve however many EfileSubmission records there are in the DB and then discard them in Rubyland

It's not a big deal since any given client has <4 TaxReturn records, but if we were doing this in a paginated list or something it wouldn't be ideal

[I think the SQLier way is more like](https://stackoverflow.com/a/35508839)

`@client.tax_returns.includes(:efile_submissions).where.not(efile_submissions: { id: nil })`
or
`@client.tax_returns.joins(:efile_submissions).distinct`